CMakeLists.txt

```diff
@@ -9,13 +9,15 @@ project(Orbit C CXX)
 if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
   set(STRICT_COMPILE_FLAGS -Werror=all
                            -Werror=float-conversion
+                           -Werror=format-security
```

Comment:
-Wformat is part of -Wall?

Reply:
Yes, but -Wformat is equivalent to -Wformat=1. And -Wformat=1 does NOT include -Wformat-security. We need to specify -Wformat=2 to get -Wformat-security. And we probably want to do that anyway. Should we do it now?

Reply:
yes if it does not involve too much effort.

Reply:
Alright. I'm on it. I don't expect too much of a hassle. If I remember correctly you pretty much removed all c-style printfs.

Reply:
Ok, it's done. I had to remove the AddLog functions in OrbitGl which haven't been used anyway. Or let's say the formatting feature was not needed. PTAL!